### PR TITLE
configure sbt-buildinfo to generate a java class

### PR DESCRIPTION
**Pull request by damdev**
Fix to #26

### CODE REVIEW

File: project/JavaClassBuildInfoRender.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+case class JavaClassBuildInfoRender (options: Seq[BuildInfoOption], pkg: String, obj: String) extends BuildInfoRenderer {
+  override def fileType = BuildInfoType.Source
+
+  override def extension = "java"
+
+  override def isSource = true
+
+  override def renderKeys(infoKeysNameAndValues: Seq[BuildInfoResult]) =
+    infoKeysNameAndValues.map { t => s"""  public static final ${t.typeExpr} ${t.identifier} = "${t.value}";""" } ++
+    infoKeysNameAndValues.map { t => s"""  public static final ${t.typeExpr} ${t.identifier}() { return ${t.identifier}; }""" }
```
Comment (ivantopo) [inline project/JavaClassBuildInfoRender.scala]: no setters? :joy:

Comment (damdev) [inline project/JavaClassBuildInfoRender.scala]: jaja, should be a spring bean??

Comment (ivantopo) [inline project/JavaClassBuildInfoRender.scala]: with extra AbstractProxyDelegatingDispatcherFactory, please :)

**Comment by dpsoft** [comment]
@damdev thanks for proposing this PR!!

**Pull request merged**